src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -1256,7 +1256,7 @@ private void VariableSummaries(RefVariable var)
             Tensor logits = null;
             tf_with(tf.name_scope(layerName), scope =>
             {
-                RefVariable layerWeights = null;
+                ResourceVariable layerWeights = null;
```

Comment:
@Oceania2018 
Is the replacement correct here?

Reply:
Should use `IVariableV1` interface.